behat_suites.yml

```diff
@@ -35,6 +35,8 @@ adminui:
                 - EzSystems\EzPlatformAdminUi\Behat\BusinessContext\DashboardContext
                 - EzSystems\BehatBundle\Context\Object\ContentTypeContext
                 - EzSystems\BehatBundle\Context\Api\TestContext
+                - EzSystems\EzPlatformAdminUi\Behat\BusinessContext\SearchContext
```

Comment:
I'd continue grouping the Context classes listed here by the bundle they come from

Reply:
Done, thx